src/features/rewards/walletWrapper/style.ts

```diff
@@ -297,8 +300,8 @@ export const StyledDateText = styled<StyledProps, 'span'>('span')`
 `
 
 export const StyledButton = styled<StyledProps, 'div'>('div')`
-  width: 88px;
-  margin: 12px 122px 15px 125px;
+  width: ${p => p.type === 'backupWallet' ? 101 : 88}px;
```

Comment:
can we create more general fix? Also I would remove fixed width as this will not work with i18n